core/src/com/mygdx/game/communication/Synchronizer.kt

```diff
@@ -8,5 +8,5 @@ object Synchronizer {
     @Volatile
     var cmds = mutableListOf<Command>()
     @Volatile
-    var gameStatus = GameStatus(listOf(), 0f)
+    var gameStatus = GameStatus(listOf(), false, 0f)
```

Comment:
przed boolowskimi parametrami zawsze dawaj nazwę, bo nie wiadomo co to za false. Przed liczbowymi w sumie też można. Coś typu GameStatus(listOf(), reset=false,  delta = 0f)